theme/less/components/logo.less

```diff
@@ -21,7 +21,7 @@ Because we can't blame anything or anyone for being too white, we came up with a
     display: inline-block;
     position: relative;
     background: white;
-    border: 1px solid @grey-50;
+    border: 1px solid @grey-100;
```

Comment:
Thank you, the `grey-50` logo border seemed like a test to check my optical skills!